cves/CVE-2014-0116.yml

```diff
@@ -45,7 +45,10 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  A fix applied in a previous update didn't account for the wildcard (*) operator.
+  By missing this operator in the appropriate blacklist for disallowed inputs, this 
+  allowed attackers to bypass the safeguards in place and craft exploits for the system.
```

Comment:
It may be a good idea to give one or two examples of exploits for this vulnerability. Great description otherwise!